google-cloud-storage/src/main/java/com/google/cloud/storage/Bucket.java

```diff
@@ -142,7 +143,7 @@ private BlobTargetOption(ObjectTargetOpt opt) {
 
     /** Returns an option for specifying blob's predefined ACL configuration. */
     @TransportCompatibility({Transport.HTTP, Transport.GRPC})
-    public static BlobTargetOption predefinedAcl(Storage.PredefinedAcl acl) {
+    public static BlobTargetOption predefinedAcl(Storage.@NonNull PredefinedAcl acl) {
```

Comment:
nit: can this not look like 
`public static BlobTargetOption predefinedAcl(@NonNull Storage.PredefinedAcl acl)`

Reply:
The `@NonNull` is applicable to a type, not the method parameter. So it needs to immediately precede the type, in this case `PredefinedAcl` which happens to be nested in `Storage`.

If the annotation is moved before Storage it will result in a compile error:
```
.../google-cloud-storage/src/main/java/com/google/cloud/storage/Bucket.java:146:66
java: scoping construct cannot be annotated with type-use annotation: @org.checkerframework.checker.nullness.qual.NonNull
```